core/templates/dev/head/domain/dashboard/ShowExplorationTitleSection.js

```diff
@@ -0,0 +1,17 @@
+oppia.directive('explorationTitleSection', function($compile) {
+  return {
+    restrict: 'E',
+    scope: {
+      exploration: '=',
+      activeExplorationId: '='
```

Comment:
I was thinking, can you make this to work with `&`? Because `=` binding seems to be unnecessary as these scope variables are not being modified inside the directive!